CHANGELOG.md

```diff
@@ -6,7 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [1.0.1] - Unreleased
 ### Added
-- Allow passing an explicit realm to `astarte_device_config_t`.
+- Add method that generate a random UUIDv4.
+- Allow passing an explicit realm to `astarte_device_config_t`. 
```

Comment:
I don't understand why this line is on the diff. Did you mistakenly add any space to that line?